src/__tests__/Cell.spec.js

```diff
@@ -100,10 +104,10 @@ describe('Cell Tests', () => {
     it('should append the update cell class to the dom node if present and cell is updated', () => {
       var updateClass = 'highlight-cell';
       testProps.column.getUpdateCellClass = () => updateClass;
-      var cellInstance = TestUtils.renderIntoDocument(<Cell {...testProps}/>);
+      var cellInstance = new PropTestUtils(Cell, testProps);
       // force update
       cellInstance.setProps({rowData: {}, selectedColumn: testProps.column});
```

Comment:
`renderIntoDocument` is still a thing, but `setProps` isn't anymore. because of the point of this test and the fact I wanted to touch it as little as possible, I put in `PropTestUtils` which is basically boilerplate but renders into document and then allows you to forcibly set props for testing as we do here.